ingests_tracker/src/test/scala/client/IngestTrackerClientTest.scala

```diff
@@ -0,0 +1,100 @@
+package client
+
+import java.time.Instant
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.Uri
+import org.scalatest.concurrent.{Eventually, IntegrationPatience}
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import uk.ac.wellcome.platform.archive.common.fixtures.HttpFixtures
+import uk.ac.wellcome.platform.archive.common.generators.IngestGenerators
+import uk.ac.wellcome.platform.archive.common.ingests.models.Ingest.Failed
+import uk.ac.wellcome.platform.storage.ingests_tracker.client.{
+  AkkaIngestTrackerClient,
+  IngestTrackerConflictError,
+  IngestTrackerUnknownError
+}
+import uk.ac.wellcome.platform.archive.common.ingests.models.Ingest.Succeeded
+import uk.ac.wellcome.platform.storage.ingests_tracker.fixtures.IngestsTrackerApiFixture
+
+class IngestTrackerClientTest
+    extends AnyFunSpec
+    with Matchers
+    with Eventually
+    with HttpFixtures
+    with IngestsTrackerApiFixture
+    with IngestGenerators
+    with IntegrationPatience {
```

Comment:
Since IngestTrackerClient is a trait, how about pulling these out into IngestTrackerClientTestCases that have to supply a `withIngestTrackerClient` fixture? Similar to what we did in the scala-storage library.